platform/jewel/markdown/core/src/main/kotlin/org/jetbrains/jewel/markdown/rendering/DefaultMarkdownBlockRenderer.kt

```diff
@@ -569,16 +574,45 @@ public open class DefaultMarkdownBlockRenderer(
         }
     }
 
+    @Deprecated(
+        message =
+            "This class function is not scalable as it relies on a pre-resolved MimeType object. " +
+                "This prevents automatic support for languages not explicitly defined in the MimeType system" +
+                "(e.g., from TextMate bundles). Use FencedCodeBlockInfo instead."
+    )
     @Composable
     internal open fun RenderCodeWithMimeType(
         block: FencedCodeBlock,
         mimeType: MimeType,
         styling: MarkdownStyling.Code.Fenced,
         enabled: Boolean,
     ) {
+        JewelLogger.getInstance("DefaultMarkdownBlockRenderer")
```

Comment:
Can you put the log in [`onFirstVisible`](https://developer.android.com/reference/kotlin/androidx/compose/ui/Modifier#%28androidx.compose.ui.Modifier%29.onFirstVisible%28kotlin.Long,kotlin.Float,androidx.compose.ui.layout.LayoutBoundsHolder,kotlin.Function0%29) so it only prints once?

Reply:
Also, you can use `[this@DefaultMarkdownBlockRenderer.]myLogger()`